config/SZBE69/symbols.txt

```diff
@@ -24140,10 +24140,10 @@ fn_802E73DC = .text:0x802E73DC; // type:function size:0x20
 fn_802E73FC = .text:0x802E73FC; // type:function size:0x90
 fn_802E748C = .text:0x802E748C; // type:function size:0x74
 fn_802E7500 = .text:0x802E7500; // type:function size:0x1D8
-fn_802E76D8 = .text:0x802E76D8; // type:function size:0x8
+fn_802E76D8_stub__7ArkFileFv = .text:0x802E76D8; // type:function size:0x8
```

Comment:
this should likely be called fn_802E76D8_ret0

it got hit pretty hard by code merging and is part of multiple classes